src/components/MainPage.js

```diff
@@ -92,7 +106,9 @@ class MainPage extends Component {
           });
           this.setState({ dataSet: dailyDataSet });
         }
-      );
+      ).then(() => {
+					scroll.scrollTo(525, {smooth: true})
```

Comment:
This will move the user to this location on a desktop browser so they can see the `Graph` now that it is populated.